client/gutenberg/extensions/map-block/add-point/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Wordpress dependencies
```

Comment:
Seems like those are external dependencies for Calypso. 

Also, mind the capital `P` in `WordPress` - WP core has [a function](https://codex.wordpress.org/Function_Reference/capital_P_dangit) to autocorrect this, but we don't :)

This applies to other occurences in this PR.

Reply:
Oops, yes will globally fix all lower p WordPress. Re: the external dependencies note are you suggesting I should remove the `Wordpress dependencies` comment block everywhere and simply move those imports to `External dependencies`?

Reply:
Moving them all to `External dependencies` and removing the `WordPress` one makes sense 👍

Reply:
Great, will do! I did pick up the pattern from other JetPack block code, e.g.: https://github.com/Automattic/wp-calypso/blob/master/client/gutenberg/extensions/tiled-gallery/editor.js#L3-L5. Would we want to make the same change everywhere?

Reply:
Yep, I've done the change in the Tiled gallery as well 👍 (pending merge) and good to PR changes anywhere else you spot it.

The pattern came from Gutenberg core blocks originally.

Reply:
Resolved in https://github.com/Automattic/wp-calypso/pull/27836/commits/a093d10a6ce2ad3116c67022db1ac1d94300d5ce